access/two_factor_authentication.md

```diff
@@ -13,15 +13,15 @@ This policy applies to all Lullabot employees and contractors.
 
 ### Explanation and Implementation
 
-Traditional authentication consists of a username and password. However, there is a more secure way to authenticate, called two-factor authentication. Two-factor authentication consists of something you know (a username and password) combined with a code available from something you have. The "something you have" is usually another device, like your phone.
+Traditional authentication consists of a username and password. However, there is a more secure way to authenticate, called two-factor authentication. Two-factor authentication consists of something you know (a username and password) combined with a code available from something you have. The "something you have" is usually a single use PIN.
 
 When logging into an account that has been set up with two factor authentication, you will be asked for your username and password as usual, but you will also be asked for the _current_ PIN. 
 
-There are several ways to generate this PIN. We strongly recommend using 1Password's [built-in 2FA functionality](https://support.1password.com/one-time-passwords/). When an authentication app is linked to your account, the app will display a six digit PIN that changes once every minute. This is preferred over using SMS for 2FA, which can by [bypassed](https://www.securityweek.com/6-ways-attackers-are-still-bypassing-sms-2-factor-authentication/).
+There are several ways to generate this PIN. We strongly recommend using 1Password's [built-in 2FA functionality](https://support.1password.com/one-time-passwords/). When an authentication app is linked to your account, the app will display a PIN, commonly 6 digits, that changes once every minute or so. This is preferred over using SMS for 2FA, which can by [bypassed](https://www.securityweek.com/6-ways-attackers-are-still-bypassing-sms-2-factor-authentication/).
```

Comment:
30 seconds seems to be more common, so I added "or so".

Since not always 6 digits, iirc, I changed to commonly 6 digits.